source/mir/ndslice/selection.d

```diff
@@ -1594,7 +1653,7 @@ template IndexSlice(size_t N)
     {
         private size_t[N-1] _lengths;
 
-        auto save() @property const
+        auto save() @property
```

Comment:
See line 1641 - it's transitive, so even `byElement` won't be possible otherwise.